docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -170,32 +328,66 @@ display:
     factor: ${x.factor}
 ----
 
-Note that you can reference nested properties using the `.` (dot) separator as in `${x.factor}`.
+[NOTE]
+====
+You can reference nested properties by using the `.` (dot) separator, as in `{x.factor}`.
+====
+
+.Additional resources
+
+* For more information about property expressions, see xref:con_property-expressions_{context}[Property expressions].
+// * For an example of property expressions using a properties file, see link:{URL_CONFIGURATION_QUARKUS}#ref_example-usage-of-property-expressions_quarkus-configuration-guide[Example usage of property expressions]. // TBD update and enable this ^ link later when this guide becomes available
+
+// DELETE meta:include::modules/con_external-application-yaml-file-for-configuring-properties-at-runtime.adoc[leveloffset=+1]
+// CREATE meta:include::modules/proc_configure-properties-at-run-time-external-yaml.adoc[leveloffset=+1]
+[id="proc_configure-properties-at-run-time-external-yaml_{context}"]
+== Configure properties at run time
```

Comment:
Don't we write it `runtime`? At least that's how we used to write it everywhere? Did it change?

Reply:
IBM Style states:
```
run time, runtime noun
- Write as two words to refer to the time at which an application runs (for example, “At run time, you can…”).
- Write as one word to refer to the environment in which an application runs (for example, “The Java runtime is
controlled by…”).
```
However, if you feel strongly about it, we can use `runtime`. Most users are probably fine with it as a single word. Just let me know.